stock_move_force_reservation/models/stock_move.py

```diff
@@ -27,7 +27,7 @@ def _compute_show_force_quantity(self):
 
     def _compute_qty_in_stock(self):
         for record in self:
-            record.qty_in_stock = record.product_id.qty_available
+            record.qty_in_stock = record.product_id.with_context({'warehouse':record.location_id.get_warehouse().id}).qty_available
```

Comment:
```suggestion
            record.qty_in_stock = record.product_id.with_context(warehouse=record.location_id.get_warehouse().id}).qty_available
```

Reply:
Moreover, why limiting the quantity to the origin location warehouse ?

Reply:
because it is for pick stock from another picking so it should be in the same warehouse so i want to know the current stock of the warehouse